admin/database/postgres/postgres.go

```diff
@@ -441,6 +441,15 @@ func (c *connection) DeleteProjectWhitelistedDomain(ctx context.Context, id stri
 	return checkDeleteRow("project whitelist domain", res, err)
 }
 
+func (c *connection) FindDeployments(ctx context.Context, createdAfter time.Time, limit int) ([]*database.Deployment, error) {
+	var res []*database.Deployment
+	err := c.getDB(ctx).SelectContext(ctx, &res, "SELECT d.* FROM deployments d WHERE d.created_on > $1 ORDER BY d.created_on LIMIT $2", createdAfter, limit)
```

Comment:
Would be better to order with `id` and paginate using  `afterID` since that's the primary key and indexed

Reply:
My understanding was that given our page size selects a significant number of rows in the table the index will not be useful and postgres will sort entire table(https://www.postgresql.org/docs/current/indexes-ordering.html). Given UUID is not naturally sortable it is better to sort on created_on. 
I could also confirm both by running explain on production table. Though deployments are supposed to increase but was not sure what that ratio would be.

But as you mentioned creation time is not unique so it is indispensable to paginate on id now. I have made the relevant changes.